src/SolverAPI.jl

```diff
@@ -558,3 +563,79 @@ function solveCliqUp!(fg::AbstractDFG,
 
   return hist, beliefMessageOut
 end
+
+"""
+Internal function used for solveCliqDown!
```

Comment:
Perhaps just specify Tx or Rx at this `cliq` vs `parent`

Reply:
I'm not following this comment, do you want clearer comments on whether it is an out or in message?

Reply:
put is Tx and take! is Rx